packages/@react-spectrum/test-utils/package.json

```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
+    "@react-aria/interactions": "3.1.0",
```

Comment:
Created a constant in useLongPress for long press threshold which is used in MenuTrigger test-case